package.json

```diff
@@ -64,6 +64,7 @@
     "react-grid-layout": "^1.2.4",
     "react-helmet": "^6.1.0",
     "react-intl": "^2.7.2",
+    "react-joyride": "^2.3.0",
```

Comment:
The pipeline failed because Snyk detected a [vulnerability](https://snyk.io/test/npm/react-joyride/2.3.0) with `react-joyride` v2.3.0. I _think_ we can still merge it, but it might be a good idea to ask @shayc to add an exception in Snyk.

Reply:
@sylvansson I added you to Snyk as a member.. you should have all the permissions. If you want I can take a look at it in a few days and add the exception.

Reply:
Thanks @shayc. I actually don't see `react-joyride` in the list of vulnerabilities so it's not immediately obvious what caused the security check to fail. I'll take a closer look this weekend.